watertap/data/techno_economic/nanofiltration.yml

```diff
@@ -0,0 +1,92 @@
+default:
+  removal_mass_solute:
+    eeq:
+      constituent_longform: Estradiol Equivalency (EEQ)
+      units: kg/kg
+      value: 0.75
+    endothall:
+      constituent_longform: Endothall
+      units: kg/kg
+      value: 0.8
+    endrin:
+      constituent_longform: Endrin
+      units: kg/kg
+      value: 0.8
+    hardness:
+      constituent_longform: Hardness
+      units: kg/kg
+      value: 0.97
+    hydroxide_alkalinity_as_caco3:
+      constituent_longform: Hydroxide Alkalinity as CaCO3
+      units: kg/kg
+      value: 0.97
+    iron:
+      constituent_longform: Iron
+      units: kg/kg
+      value: 0.8
+    magnesium:
+      constituent_longform: Magnesium, Dissolved
+      units: kg/kg
+      value: 0.99
+    manganese:
+      constituent_longform: Manganese
+      units: kg/kg
+      value: 0.8
+    ndma:
+      constituent_longform: N-nitrosodimethylamine (NDMA)
+      units: kg/kg
+      value: 0.05
+    nonvolatile_toc:
+      constituent_longform: Nonvolatile TOC
+      units: kg/kg
+      value: 0.93
+    perfluorooctane_sulfonate_pfos:
+      constituent_longform: Perfluorooctane sulfonate (PFOS)
+      units: kg/kg
+      value: 0.9
+    perfluorootanoate_pfoa:
+      constituent_longform: Perfluorootanoate (PFOA)
+      units: kg/kg
+      value: 0.9
+    phosphates:
+      constituent_longform: Phosphates, Dissolved
+      units: kg/kg
+      value: 0.99
+    potassium:
+      constituent_longform: Potassium
+      units: kg/kg
+      value: 0.99
+    sodium:
+      constituent_longform: Sodium, Dissolved
+      units: kg/kg
+      value: 0.97
+    sulfate:
+      constituent_longform: Sulfate, Dissolved
+      units: kg/kg
+      value: 0.97
+    sulfur:
+      constituent_longform: Sulfur
+      units: kg/kg
+      value: 0.97
+    toc:
+      constituent_longform: Total Organic Carbon (TOC)
+      units: kg/kg
+      value: 0.75
+    tss:
+      constituent_longform: Total Suspended Solids (TSS)
+      units: kg/kg
+      value: 0.97
+    ultraviolet_transmittance_uvt:
+      constituent_longform: Ultraviolet Transmittance (UVT)
+      units: '%'
+      value: 90.0
+  electricity_intensity:
+    units: kWh/m^3
+    value: 0.231344952
+  recovery_vol:
+    reference: Based on Vogtle case study data. No costs for WT3 treatment.
```

Comment:
```suggestion
```

Reply:
Any reason why you think this should be removed? Note that I generally did not make changes to the contents of the yml file.

Reply:
It isn't really relevant. We're just assuming a recovery rate of 85%, so the note on "costs for WT3 treatment" doesn't need to be here. Unless @kurbansitterley says that the recovery rate value was taken from this Vogtle study which provides further insight on underlying assumptions for NF performance parameters included here in the yaml (even then, 85% is a valid recovery rate that can just be assumed for NF), it doesn't need to be here since we aren't providing references for the other parameters.

Reply:
Alternatively, if that Vogtle study was used for all or several of the parameter values in the yaml, we could just have a higher-level reference indicating that.